kernel_tuner/runners/sequential.py

```diff
@@ -100,7 +100,7 @@ def run(self, parameter_space, tuning_options):
                 params = process_metrics(params, tuning_options.metrics)
 
             # get the framework time by estimating based on other times
-            total_time = 1000 * (perf_counter() - self.start_time) - warmup_time
+            total_time = 1000 * ((perf_counter() - self.start_time) - warmup_time)    # TODO is it valid that we deduct the warmup time here?
```

Comment:
It depends on when `self.start_time` (before or after the warm-up?) is set and whatever `perf_counter()` returns. I don't know about these KernelTuner details to give a definitive answer, though.